Tasks/Common/VstsAzureHelpers_/ImportFunctions.ps1

```diff
@@ -115,18 +137,21 @@ function Import-FromSdkPath {
             $path = [System.IO.Path]::Combine($programFiles, $partialPath)
             Write-Verbose "Checking if path exists: $path"
             if (Test-Path -LiteralPath $path -PathType Leaf) {
+                if ($azurePsVersion -and (-not ($(Get-SdkVersion) -eq $azurePsVersion))) {
+                    continue
```

Comment:
Why should we continue? There can be only one SDK version installed in a box. Given that it does not match why should we continue?